tests/checkpoint/test_checkpoint.py

```diff
@@ -49,11 +50,9 @@
 
 
 @pytest.mark.unit
-def test_checkpoint_no_validation_definitions_raises_error():
-    with pytest.raises(ValueError) as e:
-        Checkpoint(name="my_checkpoint", validation_definitions=[])
-
-    assert "Checkpoint must contain at least one validation definition" in str(e.value)
+def test_checkpoint_no_validation_definitions_does_not_raise():
+    Checkpoint(name="my_checkpoint", validation_definitions=[])
```

Comment:
Not a very interesting test, but I think it's worth it; I did have to change around a bit of logic to not error out on the empty list.